README.md

```diff
@@ -53,13 +53,34 @@ persist('some', someStore, {
   - **key** *string* The key of your storage engine that you want to persist to.
   - **store** *MST store* The store to be persisted.
   - **options** *object* Additional configuration options.
-    - **storage** *[localForage](https://github.com/localForage/localForage) / AsyncStorage / localStorage* [localForage](https://github.com/localForage/localForage)-style storage API. localStorage for Web (default), AsyncStorage for React Native.
+    - **storage** *[localForage](https://github.com/localForage/localForage) / AsyncStorage / localStorage*
+      Any Storage Engine that has a Promise-style API similar to [`localForage`](https://github.com/localForage/localForage).
+      <br>
+      The default is `localStorage`, with a built-in adapator to make it support Promises.
+      For React Native, one may configure `AsyncStorage` instead.
+      <br>
+      Any of [`redux-persist`'s Storage Engines](https://github.com/rt2zz/redux-persist#storage-engines) should also be compatible with `mst-persist`.
     - **jsonify** *bool* Enables serialization as JSON (default: `true`).
     - **whitelist** *Array\<string\>* Only these keys will be persisted (defaults to all keys).
     - **blacklist** *Array\<string\>* These keys will not be persisted (defaults to all keys).
 
 - returns a void Promise
 
+### Node and Server-Side Rendering (SSR) Usage
+
+Node environments are supported so long as you configure a Storage Engine that supports Node, such as [`redux-persist-node-storage`](https://github.com/pellejacobs/redux-persist-node-storage), [`redux-persist-cookie-storage`](https://github.com/abersager/redux-persist-cookie-storage), etc.
+This allows you to hydrate your store server-side.
+
+For SSR though, you may not want to hydrate your store server-side, so it is recommended that you only call `persist` conditionally:
```

Comment:
"it is recommended that you only" -> "in that case you can"

It's not recommended per se, it's an option.